setup.cfg

```diff
@@ -61,6 +61,7 @@ install_requires =
     sqlalchemy
     semver
     markdownify
+    jsonlines
```

Comment:
Havine a dependency for jsonlines is an overkill. We need to keep deps to the minimum.